app/mobile_app/mobile_app_registry.py

```diff
@@ -1,35 +1,27 @@
 from typing import List
-from flask import current_app
-from .mobile_app_types import MobileAppType
+
 from .mobile_app import MobileApp
+from .mobile_app_types import MobileAppType
 
 
 class MobileAppRegistry:
-    _instance = None
```

Comment:
Does not actually make class a Singleton in the scope of the entire application.